internal/commands/gen/tfenv.go

```diff
@@ -184,3 +203,22 @@ func GenerateTerraformFiles(project *config.Project, terraformStateBucketName st
 
 	return nil
 }
+
+func checkTFState(sess *session.Session, name string) bool {
```

Comment:
This function seems to be checking the existence of a lock but not S3 bucket.

The goal is to check if bucket exists (old code) and use that. If the bucket doesn't exist - use new defaults.
If it is set via parameters then use that value.

Does it make sense?